src/otx/v2/adapters/torch/lightning/modules/models/backbones/tiny_vit.py

```diff
@@ -0,0 +1,837 @@
+"""TinyViT for MobileSAM."""
+
+# Copyright (c) 2022 Microsoft
```

Comment:
@yunchu What should we do about the licensing of these files?
@sungchul2 Could you answer the second one?

Reply:
I checked vit_b implemented in otx (=official SAM), timm, and torchvision), and they seem to have so different architectures. Since it is not usable by simply matching the checkpoint names of the layers, it seems best to implement the model as it is now.